CHANGELOG.rst

```diff
@@ -4,6 +4,37 @@ Changelog
 1.0.0b5 (unreleased)
 ++++++++++++++++++++
 
+Features:
+
+- ``apispec.core.Components`` is added. Each ``APISpec`` instance has a
+  ``Components`` object used to define components such as schemas, parameters
+  or reponses. "Components" is the OpenAPI v3 terminology for those reusable
+  top-level objects.
+- ``apispec.core.Components.parameter`` and ``apispec.core.Components.response``
+  are added.
+- *Backwards-incompatible*: ``apispec.APISpec.add_path`` and
+  ``apispec.APISpec.add_tag`` are renamed to ``apispec.APISpec.path`` and
+  ``apispec.APISpec.tag``.
+- *Backwards-incompatible*: ``apispec.APISpec.definition`` is moved to the
+  ``Components`` class and renamed to ``apispec.core.Components.schema``.
+
+::
+
+    # apispec<1.0.0b5
+    spec.add_tag({'name': 'Pet', 'description': 'Operations on pets'})
+    spec.add_path('/pets/', operations=...)
+    spec.definition('Pet')
+    def PetSchema(Schema)
```

Comment:
The `def PetSchema(Schema)` is a bit confusing. Remove?

Reply:
Oops. Right. I got mixed up with flask-rest-api where I use this as a decorator around `Schema` definitions.